INSTALL-UBUNTU.md

```diff
@@ -0,0 +1,257 @@
+# Objective
+Install the FediBuzz relay using a virtual machine running Ubuntu Server 22.04.4 LTS. If you are familiar with NixOS/Flakes, then [installing the NixOS module](https://github.com/astro/buzzrelay?tab=readme-ov-file#build) for this is by far the easier route! 
+
+But for those of us unfamiliar with NixOS, here's another option. 
+
+# Server Configuration
+My system consisted of the following:
+
+* Router Configured to forward 80, 443 through a firewall to NGINX Proxy Manager
```

Comment:
That is all nice but not really relevant for a guide here. We don't want to project to people that they need proxmox or ngonx proxy manager